### ISSUE TITLE
Fastq Groomer sometimes is installed with Python 3 in Conda

### ISSUE THREAD

**Issue by jmchilton** [galaxyproject/tools-devteam#474]
But the wrapper script doesn't support it. At least this is what I suspect is happening.

I have an e-mail from a researcher at the NIH which states:

> One of the tools I use is the Fastq groomer which I have installed through the tool shed. I have searched a lot for the source of this error and I can’t find any solution. Every time I run this tool, I get this:

``` 
File “…/shed_tools/tooldshed.g2.bx.psu.edu/repos/devteam/fastq_groomer/e4d28c94242/fastq_groomer/fastq_groomer.py”, line 29
Print “Groomed %i %s reads into %s.” % (read_count + 1, input_type, output_type)
 
SyntaxError: Invalid syntax
 ```
 
> I am currently running python 2.7.8 and I did not find a: from __future__ import print_function anywhere so I am confused as why this error continually appears.

I suspect finishing up and merging https://github.com/galaxyproject/tools-devteam/pull/449 should fix this problem since that uses an update galaxy sequence utils with Python 3 support.

**Comment by mvdbeek** [comment]
Surely we should fix this, but grooming should only ever be necessary with very old datasets (I believe Illumina changed their pipeline in February 2011, see https://en.wikipedia.org/wiki/FASTQ_format#Encoding). That should be mentioned more prominently in the wrapper.

**Comment by nsoranzo** [comment]
Fixed with #449.
